### ISSUE TITLE
Feature: Add dark mode option

### ISSUE THREAD

**Issue by nicolettepawar** [nicolettepawar/MBA-Tech-Demo#2]
Users should be able to switch between light and dark themes. Helps reduce eye strain and improves accessibility.
